charts/sptribs-frontend/values.preview.template.yaml

```diff
@@ -2,6 +2,8 @@ nodejs:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
```

Comment:
should be defaulted to false in values.template.yaml?

Reply:
No we want to override it in flux - always enabled by default